packages/store/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@fuels/xstore",
```

Comment:
As this package also requires react we can change the name for `@fuels/react-xstore`

Reply:
✅